api/present_error.go

```diff
@@ -43,10 +45,10 @@ func presentError(ctx context.Context, c *gin.Context, err error) bool {
 
 	case errors.Is(err, context.DeadlineExceeded):
 		logger.WarnContext(ctx, fmt.Sprintf("Deadline exceeded: %v", err))
-		c.JSON(http.StatusRequestTimeout, errorResponse)
```

Comment:
If the API times out, I don't want to leak logic of where it happened exactly in the API response (though TBH our code is public so it's not as bad as it could be).